tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1

```diff
@@ -2,21 +2,14 @@
   QuickStart script for setting up Azure for AKS Edge Essentials and deploying the same on the Windows device
 #>
 param(
-    [ValidateNotNullOrEmpty()]
-    [String] $SubscriptionId,
-    [ValidateNotNullOrEmpty()]
-    [String] $TenantId,
-    [ValidateNotNullOrEmpty()]
-    [String] $Location,
-    [ValidateNotNullOrEmpty()]
-    [String] $ResourceGroupName,
-    [ValidateNotNullOrEmpty()]
-    [String] $ClusterName,
-    [String] $CustomLocationOid,
+    [Parameter(Mandatory)]
```

Comment:
Please keep the existing parameter set and add this as a separate parameter set. Don't want this as a breaking change for QuickStart flow.

Reply:
This will not be a breaking change as it is only used for installation/deployment. Let's make it clear in documentation, there isn't much benefit to keeping the old parameters...

Reply:
agree for AIO scenario. Thanks